apps/platform-app/src/components/LoginBtn/LoginBtnsWrapper.tsx

```diff
@@ -5,8 +5,8 @@ const LoginBtnsWrapper = () => {
   return (
     <div className={styles.wrapper}>
       <LoginBtn variant="secondary">Login</LoginBtn>
-      <div className={styles.btnSeparator}></div>
-      <LoginBtn variant="secondary">Join as Hunter</LoginBtn>
+      {/* <div className={styles.btnSeparator}></div> */}
```

Comment:
**nit:** Why removed / commented?

Reply:
Beaucouse now we have new design in Figma.